e2e/libs/setting/setting.py

```diff
@@ -26,26 +26,46 @@ def __init__(self):
         self.retry_count, self.retry_interval = get_retry_count_and_interval()
         self.backupstore = BackupStore()
 
+    def _update_setting(self, key, value):
+        setting = get_longhorn_client().by_id_setting(key)
+        get_longhorn_client().update(setting, value=value)
+
     def update_setting(self, key, value, retry=True):
```

Comment:
Could we know what the improvement were made to the `update_setting` function? It seems we don't need to change anything in this function as it already fulfills the requirements of the test cases you're trying to add.

Reply:
> Could we know what the improvement were made to the update_setting function?

`update_setting` shouldn't include any validation logic.

Reply:
> Could we know what the improvement were made to the update_setting function?

`update_setting` shouldn't include any validation logic.
Based on its literal meaning, it should only perform the update without any validation, to avoid misleading the upper-level callers.